recipes/cyclus/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "osicbc" %}
+{% set sha256 = "577c9a43506d1e171c80c9b6132ba2ed2a91d4d448c9c1e78d6fb794301536ef" %}
+{% set variant = "openblas" %}
+
+package:
+  name: cyclus
+  version: {{ version }}
+
+source:
+  fn: cyclus-{{ version }}-{{ sha256 }}.tar.gz
+  url: https://github.com/scopatz/cyclus/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  detect_binary_files_with_prefix: true
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - python
+    - cmake
+    - pkg-config
+    - libxmlpp
+    - coincbc
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - boost 1.61.*
+    - hdf5 1.8.17|1.8.17.*
+    - sqlite 3.13.*
+    - pcre
+    - gettext
+    - bzip2 1.0.*
+    - xz 5.2.*
+  run:
```

Comment:
I see. NVM then.

Was just noticing that `conda-build-all` is treating this like there are multiple builds to do (one for each Python version). However, they were all named the same as Python was not a run dependency. If Python was a run dependency, then we should add it. However, if it is not, we should skip those other Python versions.